src/wdio/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.4'
+
+services:
+  selenium_hub:
+    image: 'selenium/hub:latest'
+    environment:
+      GRID_THROW_ON_CAPABILITY_NOT_PRESENT: 'false'
+      GRID_MAX_SESSION: 10
+      GRID_BROWSER_TIMEOUT: 100000
+      GRID_TIMEOUT: 90000
+      GRID_NEW_SESSION_WAIT_TIMEOUT: 300000
+    ports:
+      - '4444:4444'
+  chrome:
+    image: 'selenium/node-chrome:latest'
+    volumes:
+      - /dev/shm:/dev/shm
+    ports:
+      - '5555:5555'
+    deploy:
+      replicas: 5
+    environment:
+      HUB_PORT_4444_TCP_ADDR: selenium_hub
+      HUB_PORT_4444_TCP_PORT: 4444
+      NODE_MAX_SESSION: 1
+      TZ: 'America/Chicago'
```

Comment:
Including america/chicago timezone by default. Once we move this out into a standalone project, we'll have toolkit provide its own compose file which will include this setting for terra test runs